interactive_templates/templates/v2/analysis/plot_measures.py

```diff
@@ -21,6 +22,15 @@ def main():
     df = pd.read_csv(
         f"{ args.output_dir }/joined/measure_all.csv", parse_dates=["date"]
     )
+
+    # subset of the measures file that is used for plotting in this script
+    df = df.loc[~(df["group"].isin(["event_1_code", "event_2_code", "practice"])), :]
+
+    Path(f"{args.output_dir}/for_checking").mkdir(parents=True, exist_ok=True)
```

Comment:
Not a necessary change, but if you are using the pathlib library, you don't have to use format strings
So you could specify from the start that the argparse output dir argument is type=Path
Then do 
`(args.output_dir / "for_checking").mkdir(parents=True, exist_ok=True)`
and then pandas.read_csv can accept a path object rather than a string